src/components/map/WmtsService.js

```diff
@@ -214,7 +214,8 @@ goog.require('ga_urlutils_service');
               layerIdentifier, options.capabilitiesUrl);
           if (layerOptions) {
             layerOptions.opacity = options.opacity || 1;
-            layerOptions.visible = options.visible || true;
+            layerOptions.visible = options.visible === false ?
+              options.visible : true;
```

Comment:
After some testing ([Test Link](https://mf-geoadmin3.dev.bgdi.ch/ltkum/index.html?lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&zoom=7&Y=694000&X=168000&layers_opacity=0.3&layers_visibility=false&layers=WMTS||AvalancheATHM||https:%2F%2Fmap.skitourenguru.ch%2FATHM_GetCapabilities.xml)), it seems like transformation of string to boolean is already done somewhere else. which means whatever you input as a String, only the string "false" will return flase. in that context, yuo can use layerOptions.visible=options.visible && true.

Reply:
Okay so, I did the change and pushed it. If it works, you can merge. I would propose testing with false, but also with some other strings (null springs to mind, but any string is fine). I did on dev, but better safe than sorry.